velox/docs/functions/presto/binary.rst

```diff
@@ -93,3 +93,7 @@ Binary Functions
 .. function:: xxhash64(binary) -> varbinary
 
     Computes the xxhash64 hash of ``binary``.
+
+.. function:: to_ieee754_32(real) -> varbinary
```

Comment:
The functions are maintained in alphabetical order in this file. Please move this to its correct location.

Reply:
Fixed. Will follow this from next time.